### PR TITLE
Pull Request for babyV keyboard

### PR DESCRIPTION
**Pull request by joedinkle**
## Description

Adding babyV keyboard to QMK
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/eyeohdesigns/Readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+eye oh designs
```
Comment (zvecr) [inline keyboards/eyeohdesigns/Readme.md:1]: Should be renamed to lower case

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# babyv
```
Comment (zvecr) [inline keyboards/eyeohdesigns/babyv/readme.md:1]: This file should really follow the template <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/rules.mk>

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/eyeohdesigns/Readme.md (hunk 1 above)
Comment (joedinkle) [reply keyboards/eyeohdesigns/Readme.md]: Fixed.

File: keyboards/eyeohdesigns/babyv/readme.md (hunk 2 above)
Comment (joedinkle) [reply keyboards/eyeohdesigns/babyv/readme.md:1]: Fixed.

Comment (joedinkle) [reply keyboards/eyeohdesigns/babyv/readme.md]: I originally thought this was for the rules.mk, but saw you requested changes for the readme. Requested changes to the readme made.

**Review by drashna — APPROVED** [review]

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# %babyV%
+
+![%babyV%](https://i.imgur.com/iRUm9I5.jpg)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [%Eye Oh Designs%](https://github.com/joedinkle)
+* Hardware Supported: *babyV ver 1.00*
+* Hardware Availability: *Groupbuy ran from Nov - Dec 2020*
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Eye Oh Designs](https://github.com/joedinkle)
* Hardware Supported: babyV ver 1.00
* Hardware Availability: Groupbuy ran from Nov - Dec 2020
```

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# %babyV%
+
+![%babyV%](https://i.imgur.com/iRUm9I5.jpg)
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/readme.md:3]:
```suggestion
![babyV](https://i.imgur.com/iRUm9I5l.jpg)
```

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+*A short description of the keyboard/project*
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/readme.md:5]: Please fill this out.

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+* Hardware Supported: *babyV ver 1.00*
+* Hardware Availability: *Groupbuy ran from Nov - Dec 2020*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make eyeohdesigns/babyv:default
+
+Flashing example for this keyboard:
+
+    make eyeohdesigns/babyv:default:dfu
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/readme.md:17]:
```suggestion
    make eyeohdesigns/babyv:default:flash
```

File: keyboards/eyeohdesigns/babyv/rules.mk [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/rules.mk:22]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/eyeohdesigns/readme.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+When I'm not spending time with family, or busy with work, I make keyboards that I hope you'll love.
+
+Stay up to date with my projects on Instagram: @eyeohdesigns
+
+I post on reddit and Geekhack as joedinkle
+
+![Longbow](https://i.imgur.com/nZz3tII.jpg)
+
+![SPRH](https://imgur.com/DHR0beS.jpg)
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/readme.md:11]:
```suggestion
![SPRH](https://imgur.com/DHR0beSl.jpg)
```

File: keyboards/eyeohdesigns/readme.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+eye oh designs
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/readme.md:1]:
```suggestion
# eye oh designs
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/eyeohdesigns/babyv/readme.md (hunk 5 above)
Comment (joedinkle) [reply keyboards/eyeohdesigns/babyv/readme.md]: Fixed

File: keyboards/eyeohdesigns/babyv/readme.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
+# babyV
+
+![babyV](https://i.imgur.com/iRUm9I5l.jpg)
+
+*babyV is a 40% ortholinear take on the EM7/Alice layout.*
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/readme.md:5]:
```suggestion
babyV is a 40% ortholinear take on the EM7/Alice layout.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by ridingqwerty** [comment]
Merging as it looks like we have two approvals and zvecr's requests were addressed.

Thanks!

**Pull request merged**